XamlControlsGallery/ControlPages/RadioButtonPage.xaml

```diff
@@ -22,20 +23,18 @@
                     <ColumnDefinition />
                     <ColumnDefinition />
                 </Grid.ColumnDefinitions>
-                <StackPanel>
+                <muxc:RadioButtons Header="Options:">
                     <!-- A RadioButton group. -->
-                    <TextBlock Text="Options:" Style="{ThemeResource BaseTextBlockStyle}" />
                     <RadioButton x:Name="Option1RadioButton" Content="Option 1" Checked="Option1RadioButton_Checked" />
```

Comment:
Oh, so I can put something other than strings into the RadioButtons. @kikisaints, I definitely didn't get that from reading the spec, we'll need doc examples showing more than just x:String items.

Reply:
You could, but what I'm doing here is moving the TextBlock that used to be the "header" and moving it to the Header property of RadioButtons. I didn't add a sample here (I'm hoping that @kikisaints will) showing how to databind to RadioButtons.

Reply:
> > @jevansaks Jevan Saks FTE, can you also mark the RadioButton page as "preview" (or "new") so its tile shows up correctly on the app's home page?
> 
> How do I do that? :)

Add `"IsPreview": true,` to ControlInfoData.json near line 896

Reply:
@YuliKl This is a newer feature of RadioButtons as of their migration to the Repeater control. I'll add an example of that to the docs.